samples/mpu_test/README.rst

```diff
@@ -46,6 +46,13 @@ single thread: :file:`prj_single.conf`:
 
    $ make BOARD=v2m_beetle CONF_FILE=prj_single.conf run
 
+To build the version that allows flash write, use the supplied configuration
+file: :file:`prj_allow_flash_write.conf`:
+
+.. code-block:: console
+
+   $ make BOARD=v2m_beetle CONF_FILE=prj_allow_flash_write_<x>.conf run
+
 Sample Output
 =============
```

Comment:
I ran this sample on frdm_k64f and noticed the console output doesn't match what is documented in the README.rst.  Looks like perhaps the shell got added after the doc was written? Should probably fix in a follow up patch